src/base/static/map/index.js

```diff
@@ -0,0 +1,520 @@
+/* global $ Shareabouts */
+
+import MapProvider from "map";
+
+var Util = require("../js/utils.js");
+
+var LayerView = require("mapseed-layer-view");
+
+import { map as mapConfig } from "config";
+
+class MainMap {
```

Comment:
I think this is a reasonable move for now, but we should eventually refactor this Map lib into a component. MapProvider seems like a reasonable abstraction as a module, but this Map module contains a lot of view logic that would be best captured as a React component. If so, we should move it into an organism.

I also understand that `mapView` is getting pass around all over the place. Do we have a plan on how to refactor that when we refactor our map into a component?

Reply:
I think the map fits best as an organism. It feels too important to be a util.

Does it need to be a component though? All the view logic that it's responsible for is (or should) go through the mapping provider, so there's nothing to render. Could it live in the `organisms` directory as is?

> I also understand that mapView is getting pass around all over the place. Do we have a plan on how to refactor that when we refactor our map into a component?

I haven't looked at every place it's passed yet, but I was thinking the map could just be imported where it's needed?

Reply:
I would consider this a lib, but eventually will be refactored into a component. It shouldn't be classified as a organism until it functions as a component first.

We don't have to make it a component in this PR, but eventually we should refactor it into a component. It seems a bit involved to refactor this into a proper component though.